src/core/grid_based_algorithms/LbWalberlaBase.hpp

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright (C) 2019-2020 The ESPResSo project
+ *
+ * This file is part of ESPResSo.
+ *
+ * ESPResSo is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * ESPResSo is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#ifndef LB_WALBERLA_BASE_HPP
+#define LB_WALBERLA_BASE_HPP
+
+#include "config.hpp"
+
+#ifdef LB_WALBERLA
+#include <utils/Vector.hpp>
+
+#include <boost/optional.hpp>
+
+#include <cstddef>
+#include <string>
+#include <utility>
+#include <vector>
+
+/** Class that runs and controls the LB on WaLBerla
+ */
+class LbWalberlaBase {
```

Comment:
this class is not following our naming conventions as described in https://github.com/espressomd/espresso/wiki/Naming-conventions-and-folder-structure#user-defined-types